modules/deploy/pages/redpanda/manual/resilience/emergency-shadowing.adoc

```diff
@@ -0,0 +1,212 @@
+
+Redirect your applications to the shadow cluster:
+
+1. **Update connection strings** in your applications to point to shadow cluster brokers
+2. **Update DNS records** if using DNS-based service discovery
+3. **Restart applications** to pick up new connection settings
+4. **Verify connectivity** from application hosts to shadow cluster
+
+=== Step 7: Verify application functionality
+
+Test critical application workflows:
+
+[,bash]
+----
+# Verify applications can produce messages
+rpk topic produce <critical-topic> --brokers <shadow-cluster-brokers>
+
+# Verify applications can consume messages
+rpk topic consume <critical-topic> --brokers <shadow-cluster-brokers> --num 1
+----
+
+**Test**:
+
+* Message production and consumption
+* Consumer group functionality
+* Critical business workflows
+
+=== Step 8: Clean up and stabilize
+
+Once applications are running normally:
+
+[,bash]
+----
+# Optional: Delete the shadow link (no longer needed)
+rpk shadow delete <disaster-recovery-link-name> --no-confirm
+----
+
+**Document**:
+
+* Time of failover initiation and completion
+* Applications affected and recovery times
+* Data loss estimates based on replication lag
+* Issues encountered during failover
+
+== Troubleshooting common issues
+
+=== Shadow link in PAUSED state
+
+**Problem**: Shadow link shows `PAUSED` instead of `ACTIVE`
+
+**Solution**:
+[,bash]
+----
+# Resume the shadow link
+rpk shadow resume <disaster-recovery-link-name>
+
+# Wait for it to return to ACTIVE state
+rpk shadow status <disaster-recovery-link-name>
+----
+
+=== Topics stuck in FAILING_OVER state
+
+**Problem**: Topics remain in `FAILING_OVER` state for extended periods
+
+**Solution**:
+
+* Check shadow cluster logs for errors
+* Ensure sufficient resources (CPU, memory, disk) on shadow cluster
+* Verify network connectivity between shadow cluster nodes
+
+=== Application connection failures
+
+**Problem**: Applications cannot connect to shadow cluster after failover
+
+**Solution**:
+
+* Verify shadow cluster broker endpoints are correct
+* Check security group and firewall rules
+* Confirm authentication credentials are valid for shadow cluster
+* Test network connectivity from application hosts
+
+=== Consumer group offset issues
+
+**Problem**: Consumers start from beginning or wrong positions
+
+**Solution**:
+
+* Verify consumer group offsets were replicated (check your filters)
+* Use `rpk group describe <group-name>` to check offset positions
+* If necessary, manually reset offsets to appropriate positions
+
+== Recovery planning
+
+After successful failover:
+
+1. **Assess primary cluster**: Determine root cause of the outage
+2. **Plan recovery**: Decide whether to restore primary cluster or establish new primary
+3. **Data synchronization**: Plan how to synchronize any data produced during failover
+4. **Reverse replication**: Consider setting up shadowing in the opposite direction
+
+== Post-incident actions
+
```

Comment:
In general, here (and in other places in these files) there should be some introductory sentence before a list.